controller_manager/scripts/controller_group

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# Provides quick access to the services exposed by controller_manager_msgs
+import rospy
+from argparse import ArgumentParser
+from controller_manager import controller_manager_interface
+from controller_manager_msgs.srv import ListControllers, ListControllersRequest
+
+
+class Groups(object):
+    @staticmethod
+    def list(args):
+        groups = rospy.get_param('/controller_groups', {})
```

Comment:
IMO this is fine for now, but down the road it might be desirable to have this be configurable somehow, possibly an arg which allows passing the name of the group param?

Reply:
Yeah. Also it can be a namespace? CombineRobotHW is loading `robot_hardware` from the node's namespace. I don't have much time available, so I would leave it for now.

Reply:
Using `ROS_NAMESPACE` could be nice because then you can do it once in your terminal and the ergonomics of the CLI usage are otherwise preserved (not needing an extra flag all the time). And you can get that pretty much for free I think by simply removing the leading slash from the param here.

Anyway, apart from that there isn't a single obvious way forward with this; since we don't have an immediate need, I'd just leave it.